src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -740,41 +753,42 @@ export class OpenSheetMusicDisplay {
     /** Enable or disable (hide) the cursor.
      * @param enable whether to enable (true) or disable (false) the cursor
      */
-    public enableOrDisableCursor(enable: boolean): void {
+    public enableOrDisableCursors(enable: boolean): void {
         this.drawingParameters.drawCursors = enable;
         if (enable) {
-            // save previous cursor state
-            const hidden: boolean = this.cursor?.Hidden;
-            const previousIterator: MusicPartManagerIterator = this.cursor?.Iterator;
-            this.cursor?.hide();
-
-            // check which page/backend to draw the cursor on (the pages may have changed since last cursor)
-            let backendToDrawOn: VexFlowBackend = this.drawer?.Backends[0];
-            if (backendToDrawOn && this.rules.RestoreCursorAfterRerender && this.cursor) {
-                const newPageNumber: number = this.cursor.updateCurrentPage();
-                backendToDrawOn = this.drawer.Backends[newPageNumber - 1];
-            }
-            // create new cursor
-            if (backendToDrawOn && backendToDrawOn.getRenderElement()) {
-                this.cursor = new Cursor(backendToDrawOn.getRenderElement(), this);
-            }
-            if (this.sheet && this.graphic && this.cursor) { // else init is called in load()
-                this.cursor.init(this.sheet.MusicPartManager, this.graphic);
-            }
+            for (let i: number = 0; i< this.cursorsOptions.length; i++){
```

Comment:
space between "i<". shouldn't the linter catch this?

Reply:
done also